test/apps/iodemo/io_demo.cc

```diff
@@ -617,7 +630,8 @@ class DemoClient : public P2pDemoCommon {
                     }
 
                     report_performance(total_iter - total_prev_iter,
-                                       curr_time - prev_time, info);
+                                       curr_time - prev_time, info,
+                                       opts().server_addrs[conn_num]);
```

Comment:
should not print server name, since it will mix many connections in the send loop

Reply:
@yosefe Fixed.